malcolm/core/numbermeta.py

```diff
@@ -15,7 +15,7 @@
 @Serializable.register("malcolm:core/ULong:1.0", numpy.uint64)
```

Comment:
This goes away and becomes "malcolm:core/NumberMeta:1.0", the dtype switch moves back into from_dict()

Unfortunately...